source/contracts/Controlled.sol

```diff
@@ -45,9 +45,9 @@ contract Controlled is IControlled {
     function suicideFunds(address _target, ERC20Basic[] _tokens) public onlyControllerCaller returns(bool) {
         // Transfer tokens to target
         for (uint256 i = 0; i < _tokens.length; i++) {
-            ERC20Basic token = _tokens[i];
-            uint256 balance = token.balanceOf(this);
-            token.transfer(_target, balance);
+            ERC20Basic _token = _tokens[i];
+            uint256 _balance = _token.balanceOf(this);
+            _token.transfer(_target, _balance);
```

Comment:
I know your PR didn't introduce this bug, but this should be wrapped in a `require()`.

Reply:
Good catch, will fix